src/less/tooltip/ds6/tooltip.less

```diff
@@ -43,3 +43,41 @@ button.tourtip__close {
         .background-image-base64(@icon-close-base64-light);
     }
 }
+
+button.infotip__close {
+    &:focus,
+    &:hover {
+        svg {
+            fill: @icon-close-hover-color;
+        }
+
+        span {
+            .background-image-base64(@icon-close-hover-base64);
+        }
+    }
+}
+
+button.infotip__host.icon-btn {
+    &:focus,
+    &:hover {
+        opacity: 100;
```

Comment:
@ianmcburnie 
So I looked up the code and I noticed that actionable is supplying an opacity of 0.65 when doing focus/hover on a button. 
The question I have is would this be consistent with the rest of the widgets? One icon button uses opacity and the other we are overriding and using a blue background. Maybe we can have a modifier to trigger each kind? At least for SVGs?
Should we also talk with design about what we should do since this has a different flow from the normal icon button?

Reply:
Actionable is pretty old now. It's actually more of a leftover from DS4. We haven't reviewed it for a while. I imagine it will need updating to remove opacity, but let's not worry about it for now.